packages/server/src/utilities/rowProcessor/attachments.ts

```diff
@@ -34,7 +35,7 @@ export class AttachmentCleanup {
       return []
     }
 
-    if (type === FieldType.ATTACHMENTS) {
+    if (type === FieldType.ATTACHMENTS || type === FieldType.SIGNATURE) {
```

Comment:
I see this too many times in the code, when we should probably have some helpers to avoid repetitions

Reply:
We can leave this for a further clean up